### PR TITLE
Register page

### PR DESCRIPTION
**Pull request by Harshalyadav**
Solve  merge conflict

### CODE REVIEW

**Comment by Harshalyadav** [comment]
@pavankpdev are you check my PR?

**Comment by pavankpdev** [comment]
Hey @Harshalyadav apologies, I was away, I'll check your PR tomorrow.

**Comment by pavankpdev** [comment]
Hey @Harshalyadav I went through the PR, seems like the UI isn't aligned properly, [watch this video](https://www.loom.com/share/a8f9c111709d4d44ae3eb264f42b6490), I've explained the issues

File: packages/frontend/public/registerAssist/Men.png
Comment (pavankpdev) [inline packages/frontend/public/registerAssist/Men.png:1]: Do not add images to public directory, add them to `src/assets`

File: packages/frontend/src/pages/_app.tsx
Comment (pavankpdev) [inline packages/frontend/src/pages/_app.tsx:1]: Seems like you didn't take a pull from my update, please do it, I've pushed a rainbow kit setup.

File: packages/frontend/src/pages/auth/register.tsx
Comment (pavankpdev) [inline packages/frontend/src/pages/auth/register.tsx:1]: With over 500 lines of code for a single page, it seems there is room for improvement. It is advisable to carefully examine the code and consider refactoring to ensure better practices and identify any underlying issues.

File: packages/frontend/src/pages/index.tsx
Comment (pavankpdev) [inline packages/frontend/src/pages/index.tsx:1]: `pages/index.tsx` is a different route, let's not render register page here

**Review by pavankpdev — CHANGES_REQUESTED** [review]

File: packages/frontend/public/registerAssist/Men.png
Comment (Harshalyadav) [reply packages/frontend/public/registerAssist/Men.png:1]: ok

**Comment by Harshalyadav** [comment]
@pavankpdev for validating, could I use Formik, regex or you have other concept.

**Comment by pavankpdev** [comment]
> @pavankpdev for validating, could I use Formik, regex or do you have another concept?. will tablet devices show form part only?

@Harshalyadav use can use [zod](https://zod.dev/) for form validation, tablet devices will only show the forms

**Comment by Harshalyadav** [comment]
@pavankpdev please check the validation. I need your help, where I put rainbowkit wallet custom button.

**Comment by Harshalyadav** [comment]
@pavankpdev did you check my PR?

**Comment by pavankpdev** [comment]
not yet, @Harshalyadav I'll do it tonight this sunday

**Pull request merged**